### DISCUSSION TITLE
Authorship Order

### DISCUSSION THREAD

**Discussion by quaildoc** [DrMattG/TechBk1#21] [General]
Hello @DrMattG @ChloeRN @ErlendNilsen @spharsh @wblewis7 @victorianolan8 

I propose we order the authors alphabetically after the first author (me unless I go pear shaped...Is that the correct usage of this phrase Matt?). I am open to other criteria as well. Wanted to get this cleared up before we get too deep.

**Comment by ErlendNilsen** [comment]
This sounds fair to me :)

**Reply by DrMattG** [reply]
Me too :pear:

**Comment by wblewis7** [comment]
Makes sense to me.

Will
